models/user.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+const User =  mongoose.model('User', new mongoose.Schema({
+    name: {
+        type: String,
+        required: true,
+        minlength: 3,
+        maxlength: 250
+
+    },
+    userType:{
+        enum: ['Owner', 'Customer'],
```

Comment:
Better to use small letters here: ['owner', 'customer']

Reply:
ok sir